docs/static-site-generation/debugging.md

```diff
@@ -1,85 +0,0 @@
----
```

Comment:
Can we add a redirect from this page to the new debugging page in `vercel.json`?

Reply:
Yep, good idea